src/reports_update/reports_updater.rs

```diff
@@ -126,9 +126,14 @@ where
         });
 
         let measurements = exposure.measurements();
+        let report_sig_bytes: [u8; 64] = signed_report.sig.into();
 
         Ok(Alert {
-            id: format!("{:?}", signed_report.sig), // TODO this is wrong now: one report can have multiple alerts
+            id: format!(
+                "{}-{}",
+                hex::encode(report_sig_bytes.to_vec()),
```

Comment:
@i-schuetz What is the reason for using hex representation?
We can make a composite key out of report_id and contact_start.
[SO link](https://stackoverflow.com/a/734704) 
```
CREATE TABLE something (
  column1, 
  column2, 
  column3, 
  PRIMARY KEY (column1, column2)
);
```

Reply:
I thought about composite key too, but decided against it because it couples the alerts with the report internals. Though this was when we were considering using the ratchet index. `start` is a semantically valid column of `Alert`, so it may be valid in this case. Will think about it.

Re: hex, it's not related with the unique task, just an improvement. We were using previously the debug representation of the byte array (a string "[123, 4, ...]"), which occupies more space.

Reply:
Re: unique: we need the id in the app too (e.g. to retrieve linked alerts), which means that, unless we want to deal with the composite fields in the app too, we'd have to create the id string dynamically when fetching the alerts. Not difficult, but the current solution seems more straight forward.